### PR TITLE
Make the plot function the same as python function

### PR DESCRIPTION
**Pull request by zackt113**
Hi guys, just modify the R function to make it produce the same output as the python final version. Let me know if you would like to add anything. But I think it is basically doing the same stuff as the python one now. 

Zack

### CODE REVIEW

File: R/plot_ci.R [hunk 1]
```diff
@@ -12,68 +12,72 @@
...
+#' plot_ci(c(1, 2, 3, 4, 5, 6, 7), 1000, n = 100, path = "../")
 
-# library(ggplot2, quietly = TRUE)
-
-plot_ci <- function(sample, rep, bin_size = 30, n = "auto", ci_level = 0.95, 
-                    ci_random_seed = NULL, title = "Bootstrap Sampling Plot", 
-                    x_axis = "Bootstrap Sample Mean", y_axis = "Count", 
-                    save_result_to = ".") {
+plot_ci <- function(sample, rep, bin_size = 30, n = "auto", level = 0.95, 
+                    random_seed = NULL, title = "", y_axis = "Count",
```
Comment (gauthampughazhendhi) [inline R/plot_ci.R]:
```suggestion
                    seed = NULL, title = "", y_axis = "Count", 
```
In R we can call it just seed. Thanks

**Review by gauthampughazhendhi — APPROVED** [review]
@zackt113 looks good, I have only one change. Also, kindly resolve the conflict.

**Comment by zackt113** [comment]
solved the comments and conflicts! @voremargot Can you merge it if it looks good to you! Thanks!

**Review by voremargot — APPROVED** [review]
Looks great!

**Pull request merged**